README.md

```diff
@@ -1,2 +1,42 @@
-# Hello-Microverse
-first linter project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+  Hello Microverse project
+
+> In this porject we used linters and added html and css files.
+
+
+## Built With
+
+- Major languages: Html&css
+- Frameworks: Visual studio
+- Technologies used
+
```

Comment:
- [ ] You should add the `Getting Started` section and its sub-section as in the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse. You can add things like: `how to clone the repo`, `how to open it locally`, etc.